async-nats/src/connector.rs

```diff
@@ -57,6 +58,7 @@ pub(crate) struct ConnectorOptions {
     pub(crate) name: Option<String>,
     pub(crate) ignore_discovered_servers: bool,
     pub(crate) retain_servers_order: bool,
+    pub(crate) reconnect_backoff_callback: Option<CallbackArg1<usize, Duration>>,
```

Comment:
Probably

```suggestion
    pub(crate) reconnect_delay_callback: Option<CallbackArg1<usize, Duration>>,
```

Reply:
Yeah, I agree, that's a better name for it. Thanks!